src/typings/che.d.ts

```diff
@@ -115,12 +115,12 @@ declare namespace che {
     attributes: { [propName: string]: string };
   }
 
-  export interface ProjectSource {
-    location: string;
-    parameters?: {
-      [paramName: string]: any;
-    };
-    type?: string;
+  export interface ProjectV2Source {
```

Comment:
Isn't it project source used for Che Workspaces? I would like to see type used from DevWorkspace client or devfile API https://github.com/che-incubator/devworkspace-client/blob/main/src/types.ts#L80

Reply:
I have fixed it.